test/models/message_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class MessageTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
```

Comment:
yeah but is the truth really true 🤔

Reply:
Who knows, in real life this assertion is actually flaky lol